### PR TITLE
Make Locale know about the deprecated Hebrew language code.

### PR DESCRIPTION
**Pull request by Hixie**
...by making it know about ALL the deprecated language and region codes.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @HansMuller

File: lib/ui/window.dart [hunk 1]
```diff
@@ -125,14 +132,151 @@ class Locale {
...
+  ///
+  /// The primary language subtag must not be null. The region subtag is
+  /// optional.
+  ///
+  /// The values are _case sensitive_, and should match the case of the relevant
+  /// subtags in the [IANA Language Subtag
+  /// Registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
+  /// Typically this means the primary language subtag should be lowercase and
+  /// the region subtag should be uppercase.
+  const Locale(this._languageCode, [ this._countryCode ]) : assert(_languageCode != null);
```
Comment (HansMuller) [inline lib/ui/window.dart]: NICE. No more `const Locale('foo', '')

File: tools/gen_locale.dart [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
```
Comment (HansMuller) [inline tools/gen_locale.dart]: A comment that explains what this app is for (even though we may not run it very often) would be good.

**Comment by HansMuller** [comment]
LGTM

File: tools/gen_locale.dart (hunk 2 above)
Comment (Hixie) [reply tools/gen_locale.dart]: done

**Pull request merged**